resources/serverless/templates/runtimes.yaml

```diff
@@ -11,11 +11,14 @@ data:
     FROM {{ .Values.images.runtimes.nodejs12.repository}}:{{ .Values.images.runtimes.nodejs12.tag }}
     USER root
     ENV KUBELESS_INSTALL_VOLUME=/kubeless
-    
+
     COPY /src/package.json $KUBELESS_INSTALL_VOLUME/package.json
+    COPY /registry-config/* $KUBELESS_INSTALL_VOLUME/
     RUN /kubeless-npm-install.sh
     COPY /src $KUBELESS_INSTALL_VOLUME
 
+    RUN rm -rf $KUBELESS_INSTALL_VOLUME/.npmrc
```

Comment:
Yes, you're right. I added additional line.

Reply:
Actually only one config file will be added, that's why the [getPackageConfigVolumeMountsForRuntime](https://github.com/kyma-project/kyma/pull/10809/files#diff-2e2fd5a4e28ef895badfc0bf96139c52944c912be67f0b4b0b0ca254986e836dR529) function was introduced - so that we don't have to remove all unrelevant config files in every dockerfile.